src/components/InputForm.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <div>
+        <p class="errorText">{{errorText}}</p>
+        <p class="padding">Номер телефон</p>
```

Comment:
the class name should not reflect styles but the element itself. for example above we have `errorText` which is the more maintainable class name